app/controllers/access_tokens_controller.rb

```diff
@@ -35,28 +36,9 @@ def create
             user.create_token(token_hash)
             existing_user = user
         end
-        # #saving data about access and refresh tokens to database
-        # @token = Token.new(token_hash)
-        # @token.save
-
-        # #changing 'access_token' key to 'token' so RestClient gem recognizes it for API calls
-        # new_hash = {'token' => token_hash['access_token']}
-        # new_hash.merge!(token_hash.except!('access_token'))
-        # options = {'credentials' => new_hash}
-        # options.merge!(user_hash)
             
-        # puts options
-
-        #saving user info to database
-        #rememember to perform migration, add columns for user#@user = User.new(user_hash)
-        #@user.save  
+        redirect_to 'http://localhost:3000/user/index'
```

Comment:
I think you will want to redirect to a specific user's page. User index doesnt really make sense here.